### PR TITLE
Verify VCs Guide

### PR DESCRIPTION
**Pull request by angiejones**
## What type of PR is this? (check all applicable)

- [x] 🧑‍💻 Code Snippet
- [x] 📖 Content
- [x] 🧪 Tests

## Description

This PR adds a new Verify VCs guide

## Direct Link to Preview
https://deploy-preview-1109--tbd-website-developer.netlify.app/docs/web5/build/verifiable-credentials/verify-vc

## Added code snippets?
- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed

## Added tests?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed

## What gif best describes this PR or how it makes you feel?

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206190035927153
  - https://app.asana.com/0/0/1206190035927173

### CODE REVIEW

**Review by acekyd — APPROVED** [review]
Trusted and verified :D

**Review by nitro-neal — APPROVED** [review]

File: site/docs/web5/build/verifiable-credentials/verify-vc.mdx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+# Verify VCs
+
+When an entity receives the presentation of verifiable credentials (VCs), they must verify the integrity, authenticity, and content of the VCs.
+The entity performing this verification is known as the **Verifier**.
+
+
+## Verify
+
+Upon receipt, the presentation of the VCs are encoded as [JWTs](/docs/web5/build/verifiable-credentials/jwt-to-vc#what-is-a-jwt) and can be verified via the `VerifiableCredentials.verify()` function.
```
Comment (angiejones) [inline site/docs/web5/build/verifiable-credentials/verify-vc.mdx:16]:
```suggestion
Upon receipt, the presentation of the VCs is encoded as [JWTs](/docs/web5/build/verifiable-credentials/jwt-to-vc#what-is-a-jwt) and can be verified via the `VerifiableCredentials.verify()` function. 
```

File: site/docs/web5/build/verifiable-credentials/verify-vc.mdx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  }
+}
+```
+
+</details>
+
+The `areRequiredCredentialsPresent` property of the evaluation results gives a status indication of whether the requirements were met:
+
+* **info** - all required credentials are present
+* **warn** - more credentials were presented than was required
```
Comment (angiejones) [inline site/docs/web5/build/verifiable-credentials/verify-vc.mdx:164]:
```suggestion
* **warn** - more credentials were presented than were required
```

**Comment by angiejones** [comment]
> * You start with a VerifiablePresentation object, where did you get that? Should there be an page before this explaining how to create it?


yes @EbonyLouis is working on a Presentation guide. We'll link to it from here once it's ready.

**Pull request merged**